src/ukify/ukify.py

```diff
@@ -479,7 +492,9 @@ def pe_add_sections(uki: UKI, output: str):
         new_section.set_file_offset(offset)
         new_section.Name = section.name.encode()
         new_section.Misc_VirtualSize = len(data)
-        new_section.PointerToRawData = len(pe.__data__)
+        # Non-stripped stubs might still have an unaligned symbol table at the end, making their size
+        # unaligned, so we make sure to explicitly pad the pointer to new sections to an aligned offset.
```

Comment:
If you're up to the task, stripping it should be simple: The sybol table spans `PointerToSymbolTable + NumberOfSymbols * 18` and the string table comes after that and its size is located at the first 4 bytes of the string table.

(Or I'll do it once this is in)

Reply:
I'll leave this to you